tests/PySys/software-management-end-to-end/environment_sm_management.py

```diff
@@ -278,3 +377,26 @@ def check_is_installed(self, package_name, version=None):
                 ret = True
                 break
         return ret
+
+    def getpkgversion(self, pkg):
+        """ "Use apt-cache madison to derive a package version from
+        the apt cache even when it is not installed.
+        Not very bulletproof yet!!!
+        """
+        output = subprocess.check_output(["/usr/bin/apt-cache", "madison", pkg])
+
+        # Lets assume it is the package in the first line of the output
+        return output.split()[2].decode("ascii")  # E.g. "1.16-1+b3"
+
+    def get_pkgid(self, pkg):
+
+        pkgid = self.pkgiddb.get(pkg)
+
+        if pkgid:
+            return pkgid
+        else:
+            raise SystemError("Package ID not in database")
+
+    def mysmcleanup(self):
+        # Experiment
+        time.sleep(5)
```

Comment:
What are we sleeping for? We don't seem to be doing anything in this cleanup function.

Reply:
Disabled the experiment for now. Will be enabled when it helps to reduce timeouts on C8y